scripts/test/run-tests.sh

```diff
@@ -151,30 +151,15 @@ if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "ubuntu-qemux86-64-no-grub-d" ];
                    || test_result=$?
 fi
 
-if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi3" ]; then
```

Comment:
Actually, there has been a long internal discussion and we will restore raspberrypi3 this sprint. This is independent of your PR and I am okey merging yours as-is (because, well, at the time we are not building the image)